build-tools/plots-to-appinsights/ProcessPlotCSVFile.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
No need for this `.sln` file.

Instead, `build-tools/plots-to-appinsights/*.csproj` should be part of `Xamarin.Android.sln`.

Reply:
I think this is more akin to `xaprepare` which has a `xaprepare.sln` so it can be built on each test agent rather than being built in the `Mac Build` step and then included in the installer.  This tool is only used on the agent running `APK Instrumentation` so it probably makes more sense to only build it there.  

https://github.com/xamarin/xamarin-android/blob/master/build-tools/automation/yaml-templates/setup-test-environment.yaml#L14-L19

Reply:
Indeed. I pattered this after tools that build & run as part of the test jobs such as `APK Instrumentation`